src/main/target/common_pre.h

```diff
@@ -255,6 +255,9 @@
 #endif // # !defined(LED_MAX_STRIP_LENGTH)
 
 #if defined(USE_LED_STRIP)
+#if defined(STM32H7)
```

Comment:
```suggestion
#if defined(STM32H7) && !defined(USE_LEDSTRIP_CACHE_MGMT)
```

Reply:
Reason is with CLOUD BUILD people adding their own defines to the build process, without this check, that would become an error.

Reply:
Changes